quicktests/overlaying/tests/interactions/pan_zoom_reverse.js

```diff
@@ -29,6 +29,14 @@ function run(svg, data, Plottable) {
       .setMinMaxDomainValuesTo(xScale)
       .setMinMaxDomainValuesTo(yScale);
 
+    new Plottable.Interactions.PanZoom(xScale, null)
+        .attachTo(xAxis)
+        .setMinMaxDomainValuesTo(xScale);
+
+    new Plottable.Interactions.PanZoom(null, yScale)
+        .attachTo(yAxis)
+        .setMinMaxDomainValuesTo(yScale);
```

Comment:
Yeah we had discussed this - I decided against it because users can already implement that behavior by creating separate PanZoomInteractions and attaching all of them to the same plot. If we make that the behavior of PZI, then we no longer support the users who want the current "constrain all scales" behavior.

Reply:
That may be possible for plottable users, but not for our react component. If we are going to use this axis zoom ourselves, won't our users have the same problem with no way to fix it?

Reply:
We can do the same with our React Component, I was experimenting with it. We just rebuild an array of PZIs instead of a single one when the props change